src/models/Movie/type.ts

```diff
@@ -0,0 +1,20 @@
+import { Entry, EntrySkeletonType, FieldsType } from 'contentful';
+
+export type MovieModel = {
+	id: string | number;
```

Comment:
I showed you `string | number` because I didn't know what the actual type is. You should specify to match what the IDs actually are.